eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -312,6 +361,14 @@ FakeMaven withTojoAttribute(final ForeignTojos.Attribute attribute, final Object
         return this;
     }
 
+    /**
+     * Should we use defaults or not?
+     * @return The same maven instance.
+     */
+    FakeMaven withoutDefaults() {
+        return new FakeMaven(this.workspace, this.params, this.attributes, this.current, false);
```

Comment:
@volodya-lombrozo Do you think it's a good idea to return a new instance of `FakeMaven` here? It looks like in order to get `FakeMaven` without defaults you need to create first one with defaults and then call this method (like you did in test `throwsExceptionInCaseSourceDirIsNotSet `).
It's up to you but I would remove the method and uses constructor with 2 arguments in tests instead

Reply:
@maxonfjvipon To be honest, it's a good suggestion. Thank you. However, I'm trying to hide that fact that `FakeMaven` is a configurable object:
https://www.yegor256.com/2016/04/19/object-must-not-be-configurable.html

Moreover, since `FakeMaven` implements some sort of the [builder](https://en.wikipedia.org/wiki/Builder_pattern) pattern, I believe that it's convenient to keep the corresponding style.